WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -170,6 +166,17 @@ - (void)viewWillDisappear:(BOOL)animated
     self.resultsController.delegate = nil;
 }
 
+- (void)didRotateFromInterfaceOrientation:(UIInterfaceOrientation)fromInterfaceOrientation
```

Comment:
My bad, my eyes are tricking me! I was trying to use `willRotateToInterfaceOrientation: duration:` and using `willAnimateRotationToInterfaceOrientation: duration:` fixed the issue. Thanks for the heads up!